frontend/components/AddHostsModal/AddHostsModal.tsx

```diff
@@ -29,8 +32,22 @@ const AddHostsModal = ({
   onCancel,
   openEnrollSecretModal,
 }: IAddHostsModal): JSX.Element => {
+  const { isPreviewMode, config } = useContext(AppContext);
   const teamDisplayName = (isAnyTeamSelected && currentTeamName) || "Fleet";
 
+  const {
```

Comment:
Had to move this to the component level or was running into errors with useQuery being called in a subcomponent.